docs/setup/aks/getting-started-aks.md

```diff
@@ -174,16 +177,22 @@ kubectl port-forward -n nuclio $(kubectl get pods -n nuclio -l nuclio.io/app=das
 kubectl port-forward -n kube-system $(kubectl get pod -n kube-system -l k8s-app=traefik-ingress-lb -o jsonpath='{.items[0].metadata.name}') 8080:80
 ```
 
+<a id="deploy-a-function-with-the-nuclio-dashboard"></a>
 ## Deploy a function with the Nuclio dashboard
 
-Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation). You should see the [Nuclio dashboard](/README.md#dashboard) UI. Choose one of the built-in examples and click **Deploy**. The first build will populate the local Docker cache with base images and other files, so it might take a while, depending on your network. When the function deployment is completed, you can click **Invoke** to invoke the function with a body.
+Browse to `http://localhost:8070` (after having forwarded this port as part of the Nuclio installation) to see the [Nuclio dashboard](/README.md#dashboard).
+Select the "default" project and then select **New Function** from the action toolbar to display the **Create function** page (http://localhost:8070/projects/default/create-function).
+Choose one of the predefined template functions, and select **Deploy**. 
+The first build populates the local Docker cache with base images and other files, so it might take a while to complete, depending on your network.
+When the function deployment completes, you can select **Invoke** to invoke the function with a body.
 
 ## What's next?
 
 See the following resources to make the best of your new Nuclio environment:
 
-- [Deploying functions](/docs/tasks/deploying-functions.md)
-- [Invoking functions by name with an ingress](/docs/concepts/k8s/function-ingress.md)
+- [Best Practices and Common Pitfalls](/docs/concepts/best-practices-and-common-pitfalls.md)
+- [Deploying Functions](/docs/tasks/deploying-functions.md)
```

Comment:
I feel like the previous order made more sense, deploying functions sounds more basic then "best practices and common pitfalls", those things are usually left to the end of lists like this, no?

Reply:
@omesser I didn't change the order in the AKS doc, I added the best-practices reference, which was previously missing from this doc. I added it before the function-deployment doc because this is the order that Liran used in the new Docker GS doc, in PR #1708, which you had approved, so I thought that's what you guys preferred:
```markdown
- [Best Practices and Common Pitfalls](/docs/concepts/best-practices-and-common-pitfalls.md)
- [Deploying functions](/docs/tasks/deploying-functions.md)
- [More function examples](/hack/examples/README.md)
- [References](/docs/reference)
```
I certainly don't mind moving the best-practices to the end of the references list, in all the GS docs. @liranbg please confirm before I make the changes.

Reply:
I feel like it should be at the end
@liranbg  - objections?

Reply:
no, go ahead

Reply:
DONE — I moved the best-practices link to the end of all "What's next?" sections in the relevant GS guides — commit 65e4779. @omesser @liranbg please approve.